### PR TITLE
Update be and ds entries for access logs

### PR DESCRIPTION
**Pull request by ierceg**

### CODE REVIEW

File: guides/logs.md [hunk 1]
```diff
@@ -118,11 +118,11 @@ Size of the response in bytes.
 
 ### ds
 
-Destination, determined by split testing rules.
+Destination, determined by split testing rules, if there are any, otherwise left as the `default` router.
```
Comment (Steve-B-Murphy) [inline guides/logs.md:121]: Please change to the following:
Destination, determined by split testing rules, if any; if none, the value is left as the `default` router.

**Review by Steve-B-Murphy — COMMENTED** [review]
I had one suggested change.

**Comment by Steve-B-Murphy** [comment]
I had one suggested change.

**Review by jeffpatzer — APPROVED** [review]

**Comment by jeffpatzer** [comment]
@Steve-B-Murphy your change request was minor, feel free in the future to just do the change and get it merged 🙌

**Pull request merged**